Python/Easy/[423] Let's Sort This List! [E].py

```diff
@@ -71,98 +71,43 @@
 # Your code should go here:
 
 # I want to apply some sorting algorithm to it.
-def ascDesNone(lst1, sortType):
+def ascDesNone(lst1:list, sortType:str):
     # For checking if the list contains only numbers or not?
     for i in lst1:
-        if isinstance(i, (int, float)):
-            pass
-        elif not isinstance(i, (int, float)):
+        if not isinstance(i, (int, float)):
```

Comment:
The reason I am implementing such a long applications of logics is to learn to use to make programs without using any built in functions or classes.
Like just for practicing purposes.